html-forms/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+    <h1>Coding Form!</h1>
+  </head>
+  <body>
+    <div></div>
+    <label for="email">Email</label>
+    <div></div>
+    <input type="email" />
+    <div></div>
+    <label for="why-coding">Why is Coding Cool?</label>
+    <div></div>
+    <select id="custom-form">
```

Comment:
This needs an `id` attribute that matches with the `for` attribute on its `label`